modules/firemission/settings.sqf

```diff
@@ -0,0 +1,192 @@
+
```

Comment:
This file is really hard to read, even harder to understand. I recommend formatting it better (indentations, separators between different functions, etc.). At the top you can give quick summary what function to call to make it work, how to make it work, etc.